spec/services/hearings_creator_spec.rb

```diff
@@ -215,6 +250,105 @@
     end
   end
 
+  context "when an application to court" do
+    context "with a defendant that has a linked LAA reference" do
+      let(:prosecution_case_array) { nil }
+
+      it "calls the Sqs::PublishHearing service once" do
+        allow(LaaReference).to receive(:find_by).with(defendant_id: "dd22b110-7fbc-3036-a076-e4bb40d0a666", linked: true).and_return(defendant_one)
```

Comment:
we're stubbing LaaReference to return a defendant here, which is not what LaaReference actually returns, so we're at risk of having our tests pass against our stubs but not against our actual code.

Reply:
Ah, I've misunderstood something here then. The issue of expecting a Defendant object when a var is named defendant is stemming from the same misunderstanding. Will take another look. Thank you

Reply:
It just returns an ActiveRecord object `LaaReference`.